tools/ci_build/build.py

```diff
@@ -1487,24 +1487,27 @@ def generate_build_tree(
     cflags = None
     cxxflags = None
     ldflags = None
-    cudaflags = []
+    cudaflags = None
     for config in configs:
+        cflags = []
+        cudaflags = []
+        if is_windows():
+            njobs = number_of_parallel_jobs(args)
+            if njobs > 1:
+                if args.parallel == 0:
+                    cflags += ["/MP"]
+                else:
+                    cflags += ["/MP%d" % njobs]
```

Comment:
would this work for an Android build on Windows?

Reply:
Updated.